test/kontrol/README.md

```diff
@@ -0,0 +1,200 @@
+# Euler Vault Kit Kontrol Tests
+-------------------------
+
+In this repo you'll find a series of examples showcasing some of the features of Kontrol, an open-source tool for formal verification of Solidity smart contracts. 
+
+## Getting Kontrol
+
+We recommend installing Kontrol via the K framework package manager, `kup`.
```

Comment:
I'd add, after the period, " We also recommend using the latest version of Foundry."

Reply:
Done!